content/configuration/dev_authorization.md

```diff
@@ -106,7 +106,7 @@ Then, click the "Edit" link for the pipeline group you want to manage permission
 
 ![](../images/group_permission.png)
 
-If no authorization is defined for a pipeline group, all GoCD users will have view and operate permissions to that group.
+If no authorization is defined for a pipeline group, it is viewable and operable only by GoCD system administrators.
```

Comment:
Should we make this statement a little prominent. Probably add it as a note and move it above after the `admin permissions` section.

Reply:
Done. It looks like this:

------------------------

<img width="880" alt="2020_04_29_18-15-50" src="https://user-images.githubusercontent.com/172235/80626192-f7068e80-8a45-11ea-9099-b17a10c802e5.png">

------------------------